src/calculateRentalCost.js

```diff
@@ -28,7 +28,28 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  let DAYS_RATE = days * PRICE_PER_DAY;
+
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+
+  if (days >= LONG_TERM) {
+    DAYS_RATE -= LONG_TERM_DISCOUNT;
+
+    return DAYS_RATE;
```

Comment:
```suggestion
     return DAYS_RATE - LONG_TERM_DISCOUNT;
```
Just return the full price without reassigning

Reply:
👍